#! /bin/sh /usr/share/dpatch/dpatch-run
## -*- diff -*-
## termios-h-check-3.0.dpatch by Rafael Laboissiere <rafael@debian.org>
##
## DP: Small tweak for configure.in/configure to properly detect
## DP: <termios.h>, making the package build on GNU/kFreeBSD. Thanks to
## DP: Petr Salinger for the patch.  Closes Bug#520076 for octave3.0.2A

@DPATCH@
--- octave3.1-3.1.54.orig/configure.in
+++ octave3.1-3.1.54/configure.in
@@ -1247,7 +1247,7 @@
 AC_LANG_POP(C++)
 
 have_termios_h=no
-AC_CHECK_HEADERS($TERMIOS_H, have_termios_h=yes)
+AC_CHECK_HEADERS(termios.h, have_termios_h=yes)
 AC_CHECK_HEADERS(termio.h, have_termio_h=yes, have_termio_h=no)
 AC_CHECK_HEADERS(sgtty.h, have_sgtty_h=yes, have_sgtty_h=no)
 AC_CHECK_HEADERS(glob.h, have_glob_h=yes, have_glob_h=no)
